modules/standard/FileSystem.chpl

```diff
@@ -480,6 +480,33 @@ proc isLink(name: string): bool {
   return ret;
 }
 
+/* Returns true if the name corresponds to a mount point, false otherwise.
+   err: a syserr used to indicate if an error occurred
+   name: a string that could be the name of a mount point.
+*/
+proc isMount(out err:syserr, name: string): bool {
+  extern proc chpl_fs_is_mount(ref result:c_int, name: c_string): syserr;
+
+  if (isFile(name)) {
+    // Files aren't mount points.  That would be silly.
+    err = ENOERR;
+    return false;
+  }
+  var ret:c_int;
+  err = chpl_fs_is_mount(ret, name.c_str());
+  return ret != 0;
```

Comment:
This presumes chpl_fs_is_mount() always sets ret, whether or not it returns 0. But that's not true, see below: it can return 0 and not set ret in one case, and in at least 2 cases returns non-0 without setting ret. Need to either make chpl_fs_is_mount() always set *ret, or maybe 'return err == 0 && ret != 0' here?